### DISCUSSION TITLE
graphRAG only extracting information from a single column when input data format is .csv

### DISCUSSION THREAD

**Discussion by Dimsas04** [microsoft/graphrag#654] [Q&A]
I have a .csv with several columns file that I am trying to feed to graphRAG. I have set the source_column and text_column in the settings.yaml file, and have set the rest as document_attribute_columns in the same. However, when I query the model, it answers using data from the text_column exclusively. How can I make it answer using data from all the columns?

I have added the columns as entities in the entity_extraction section in the settings.yaml file as well.

Thank you in advance

**Comment by yejin2008** [comment]
I got csv file loading error.  load_input.py", line 81, in load_input
    results = await loader(config, progress, storage)
What is correct configuration for csv file
